.github/workflows/reusable-build.yml

```
@@ -101,11 +101,25 @@ jobs:
             --parallel $(nproc) \
             --target "${CMAKE_TARGET}"
 
+      - name: Put built binaries in one location
+        shell: bash
+        working-directory: ${{ inputs.build_dir }}
+        env:
+          BUILD_TYPE_DIR: ${{ runner.os == 'Windows' && inputs.build_type || '' }}
+        run: |
+          mkdir -p ./binaries/
+          mkdir -p ./binaries/doctest/
```

```suggestion
          mkdir -p ./binaries/doctest/
```

The `mkdir -p` command will create all intermediate paths.